boards.yaml

```diff
@@ -1,4 +1,66 @@
 lab-slave-0:
+  dispatcher_ip: 192.168.66.1
   boardlist:
     qemu-01:
       type: qemu
+    bcm2837-rpi-3-b-01:
+      type: bcm2837-rpi-3-b
+      pdu_generic:
+        hard_reset_command: /usr/local/bin/acme-cli -s 192.168.66.2 reset 1
+        power_off_command: /usr/local/bin/acme-cli -s 192.168.66.2 switch_off 1
+        power_on_command: /usr/local/bin/acme-cli -s 192.168.66.2 switch_on 1
+      uart:
```

Comment:
The adding of all these boards should not be part of the master branch.  This should be in a lab-specific branch someplace.

Reply:
This should never be in this PR, I have commited this by mistake sorry.